CalculatorExercises/1-CalculatorBasic/StringHelper.java

```diff
@@ -0,0 +1,18 @@
+import java.util.*;
+import java.lang.*;
+
+public class StringHelper {
+
+    public static StringBuilder removeWhiteSpace (String string) {
```

Comment:
it is Whitespace character not White Space character, please fix function name.

Reply:
also, please google 'java How to remove whitespace from String' maybe there is easier/shorter way